ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoSuite.scala

```diff
@@ -443,6 +443,24 @@ private[dao] trait JdbcLedgerDaoSuite extends AkkaBeforeAndAfterAll with JdbcLed
       implicit ec: ExecutionContext): Future[(Offset, LedgerEntry.Transaction)] =
     store(divulgedContracts = Map.empty, offsetAndTx)
 
+  protected final def storeSync(commands: Vector[(Offset, LedgerEntry.Transaction)])(
+      implicit ec: ExecutionContext): Future[Vector[(Offset, LedgerEntry.Transaction)]] = {
+
+    import scalaz.{Free, NaturalTransformation}
+    import scalaz.syntax.traverse._
+    import scalaz.std.vector._
+    import scalaz.std.scalaFuture._
+
+    val storeDelayed = (a: (Offset, LedgerEntry.Transaction)) => Free.liftF(store(a))
```

Comment:
The `suspend` from the prior suggestion is important, too.

Reply:
`def liftFU[MA](value: => MA)` this was actually taking the argument by name.... `def liftF[S[_], A](value: S[A]): Free[S, A]` does NOT... so why not to use `liftFU`

Reply:
For posterity:

```scala
scala> def boom(): Future[Int] = sys.error("boom")
boom: ()scala.concurrent.Future[Int]

scala> Free.liftF(boom())
java.lang.RuntimeException: boom
  at scala.sys.package$.error(package.scala:30)
  at .boom(<console>:15)
  ... 28 elided

scala> Free.liftFU(boom())
java.lang.RuntimeException: boom
  at scala.sys.package$.error(package.scala:30)
  at .boom(<console>:15)
  at .$anonfun$res3$1(<console>:21)
  at scalaz.Free$.liftFU(Free.scala:31)
  ... 28 elided

scala> Free.suspend(Free.liftF(boom()))
res4: scalaz.Free[scala.concurrent.Future,Int] = Gosub(Suspend(Future(Success(()))),scalaz.Free$$$Lambda$1316/150619719@205c5cd0)
```